src/vivarium/framework/time.py

```diff
@@ -145,7 +145,7 @@ def step_backward(self) -> None:
     def step_forward(self, index: pd.Index) -> None:
         """Advances the clock by the current step size, and updates aligned simulant clocks."""
         self._clock_time += self.step_size
-        if self._individual_clocks:
+        if self._individual_clocks and index.any():
```

Comment:
the .min() can be problematic with null index (helpfully discovered on first timestep of MNCH child)